package.json

```diff
@@ -10,16 +10,16 @@
     "conf dir"
   ],
   "author": "Benjamin DANIEL <benjamin.daniel@neo9.fr>",
-  "license": "GNU",
+  "license": "MIT",
   "main": "dist/src/index.js",
   "types": "dist/src/index.d.ts",
   "files": [
     "dist/src"
   ],
   "scripts": {
     "build": "rimraf dist/ && tsc",
-    "lint": "tslint --project tsconfig.json --format verbose '{src,test}/**/*.ts'",
-    "lint:apply": "npm run format && tslint --fix --project tsconfig.json '{src,test}/**/*.ts'",
+    "lint": "eslint --config .eslintrc.yaml '{src,test}/**/*.ts'",
+    "lint:apply": "npm run format && npx eslint --fix '{src,test}/**/*.ts'",
     "format": "prettier --write '{,@(src|test)/**/}*.*'",
     "test:dev": "export NODE_ENV=test && TS_NODE_FILES=true ava --verbose --color --serial --watch",
     "test": "npm run lint && npm run build && export NODE_ENV=test && nyc ava --verbose --color --serial && nyc report --reporter=html",
```

Comment:
could you do the same as here : https://github.com/neo9/n9-mongodb-client/commit/190dd532c29b49f27fbab76c657d28fff728af46#diff-e663edb91932673eb56014f795a7e6776b2080d9b54afe486a1cf0b64376b70b

remove linting from tests and add a Github Action step ?